configmap/initdb/sql/repl/02_replication_data.sql

```diff
@@ -26,7 +26,7 @@ INSERT INTO `config` VALUES ('controller', 'empty_chunks_dir',    '/qserv/data/q
 -- Database service-specific parameters
 
 INSERT INTO `config` VALUES ('database', 'services_pool_size', '32');
-INSERT INTO `config` VALUES ('database', 'qserv_master_host',               '<CZAR_DN>');
+INSERT INTO `config` VALUES ('database', 'qserv_master_host',               '{{.CzarDn}}');
```

Comment:
What is `Dn` in this context? Would it make sense to spell it out?

Reply:
It is "DomainName", I'll spell it out.